examples/jenkins/jenkins-persistent-template.json

```diff
@@ -113,6 +113,14 @@
                 },
                 "env": [
                   {
+                    "name": "OPENSHIFT_ENABLE_OAUTH",
+                    "value": "anyvalue"
+                  },
+                  {
+                    "name": "OPENSHIFT_ENABLE_REDIRECT_PROMPT",
+                    "value": "anyvalue"
+                  },
+                  {
                     "name": "JENKINS_PASSWORD",
                     "value": "${JENKINS_PASSWORD}"
```

Comment:
they're not running outside openshift if they are using the template to instantiate.

what i think we agreed to doc are two scenarios:
1) if you're running outside openshift (so you did a docker run of the image or something) the JENKINS_PASSWORD env variable should be set and is relevant.
2) if you're running inside openshift but using an older version of openshift, then oauth won't work and you'll have to login using "admin/password" (because we won't randomly generate a password anymore).  And ideally you should change the password by setting the env variable properly yourself.